tests/rbd/rbd_utils.py

```diff
@@ -919,3 +945,20 @@ def verify_migration_commit(rbd, pool, image):
     else:
         log.error((f"Image {image} is not found in pool {pool}"))
         raise ImageNotFoundError("Image Not found.")
+
+
+def verify_namespace_exist(rbd, pool, namespace):
+    """Method verifies for namespace exist in namespace list
+    Args:
+        rbd: rbd_object
+        pool: pool name
+    """
+    log.info("verifying namespcae exist in namespace list")
+    out = rbd.exec_cmd(cmd=f"rbd namespace ls --pool {pool} --format json", output=True)
+    result = json.loads(str(out))
+    for value in result:
+        if value["name"] == namespace:
+            log.info(f"namespace {namespace} is exist")
```

Comment:
Please rectify grammatically
can be matched with log.error couple of lines below

Reply:
Done.